apps/ledger-live-desktop/src/newArch/features/ModularDialog/screens/NetworkSelector/components/NetworkListItem/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { CryptoIcon } from "@ledgerhq/react-ui/pre-ldls";
+import { ListItem } from "@ledgerhq/ldls-ui-react";
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+
+export type NetworkListItemData = {
+  currency: CryptoOrTokenCurrency;
+  description?: string;
+  rightElement?: React.ReactNode;
+  apy?: React.ReactNode;
+};
+
+type NetworkListItemProps = NetworkListItemData & {
+  onClick: () => void;
+};
+
+export const NetworkListItem = ({
+  currency,
+  description,
+  rightElement,
+  apy,
+  onClick,
+}: NetworkListItemProps) => {
+  return (
+    <ListItem
+      title={currency.name}
+      leadingContent={<CryptoIcon size="48px" ledgerId={currency.id} ticker={currency.ticker} />}
+      description={description}
+      descriptionTag={apy ? <>{apy}</> : undefined}
```

Comment:
as it's already `React.ReactNode` you can remove wrapper <> no? also it handles `undefined`